addons/zeus/CfgVehicles.hpp

```diff
@@ -214,6 +214,18 @@ class CfgVehicles {
         function = QFUNC(moduleUnconscious);
         icon = QPATHTOF(UI\Icon_Module_Zeus_Unconscious_ca.paa);
     };
+    class GVAR(moduleToggleNvg): GVAR(moduleBase) {
+        curatorCanAttach = 1;
+        category = QGVAR(Utility);
```

Comment:
Shouldn't this rather be category AI? It specifically shouldn't work on players anyways (flashlight already doesn't).

Reply:
didn't really know where to put it, good point.